instrumentation/kamon-lagom/src/main/scala/kamon/instrumentation/lagom/LagomMetrics.scala

```diff
@@ -0,0 +1,49 @@
+package kamon.instrumentation.lagom
+
+import kamon.Kamon
+import kamon.metric.{Counter, Gauge, Histogram, InstrumentGroup, MeasurementUnit, Metric, Timer}
+import kamon.tag.TagSet
+
+object LagomMetrics {
+
+  val defaultTags: TagSet = TagSet.of("component", "lagom")
+
+  val CBLatency: Metric.Histogram = Kamon.histogram(
+    name = "lagom.cb.latency",
+    description = "Latency of Lagom Circuit Breakers",
+    unit = MeasurementUnit.time.nanoseconds
+  )
+
+  val CBState: Metric.Gauge = Kamon.gauge(
+    name = "lagom.cb.state",
+    description = "State of Lagom Circuit Breakers (possible values: 1 - Open, 2 - HalfOpen, 3 - Closed)"
+  )
```

Comment:
Just out of curiosity: how do you visualize this metric? I saw grafana has these [State Timeline](https://grafana.com/docs/grafana/latest/visualizations/state-timeline/) visualizations, but I'm not sure if this data would work for that.

Reply:
We use a [Gauge](https://grafana.com/docs/grafana/latest/visualizations/gauge-panel/) 😄 
But thank you for your idea about State Timeline, I think we should experiment with him.